### PR TITLE
[module] nnstreamer: Updated the existing annotations in apptest module

### PR DESCRIPTION
**Pull request by leemgs**
This commit is trivial. It is to update the existing out-of-date statements
to improve the readability and keep the maintenance.

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>

---

### CODE REVIEW

File: ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh [hunk 1]
```diff
@@ -16,14 +16,21 @@
 
 ##
 # @file    pr-audit-nnstreamer-ubuntu-apptest.sh
-# @brief   Check if nnstreamer sample applications normally work
-#          with a commit of a Pull Request (PR).
+# @brief   Check if sample applicaiton sof nnstreamer can be run.
```
Comment (jaeyun-jung) [inline ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: typo applicaiton sof

Comment (leemgs) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: Fixed

Comment (jaeyun-jung) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: maybe applications of (application sof)

Comment (leemgs) [reply ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh]: OOops. Updated. :)

**Pull request merged**